spec/figure_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require_relative '../lib/figures/figure'
+require_relative '../lib/figures/king'
+
+describe Figure do
+  subject(:king) { King.new(:white, 3, 3) }
+
+  describe '#find_possible_moves' do
```

Comment:
Your `describe` tells me that it's going to test a method called `#find_possible_moves`, but then you test `#possible_moves`. Keep your story straight